docusaurus/docs/how-to-guides/app-plugins/fetch-data-from-frontend-to-app.md

```diff
@@ -0,0 +1,132 @@
+---
+id: fetch-data-from-frontend-to-app
+title: Fetch data from frontend code to app plugin using the data proxy
+description: Learn how to use the data proxy API to fetch data from frontend code in app plugins in Grafana
+keywords:
+  - grafana
+  - plugins
+  - data proxy
+  - frontend
+  - app
+  - app plugin
+  - CORS
+---
+
+# Fetch data from app plugins
+
+Along with the JavaScript [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), the Grafana data proxy is used to fetch data from a Grafana app plugin.
+
+The data proxy is especially useful
+
+- for overcoming cross-site (CORS) limitations, or
+- for performing authenticated requests, or
+- for sending other sensitive data from your plugin configuration to Grafana.
```

Comment:
It could also be to external services/resources. E.g. if your plugin configure an authentication token that you need to use to interact with a third party API. Then you can use the data proxy to append that token before sending the request to the third party API without leaking the token to the frontend.